lambda_functions/vetext_incoming_forwarder_lambda/vetext_incoming_forwarder_lambda.py

```diff
@@ -116,26 +168,25 @@ def process_body_from_alb_invocation(event):
     event_body_encoded = event.get("body", "")
 
     if not event_body_encoded:
-        logger.info("event_body from alb record was not present")
-        logger.debug(event)
+        logger.debug("event_body from alb record was not present: %s", event)
```

Comment:
Realized this should be an error. We have no body!
```suggestion
        logger.error("event_body from alb record was not present: %s", event)
```